### ISSUE TITLE
[BUG] Irregular screen sizes

### ISSUE THREAD

**Issue by labrys** [guessit-io/guessit#276]
The `screen_size` property only uses the `B` (e.g. height) value of an `AxB` resolution when selecting named resolutions (e.g. 480p, 720p, etc).  If an oddball resolution is entered wouldn't it be more correct to return it as `AxB`?

``` python
>>> guessit('480x720')
MatchesDict([('screen_size', '720p'), ('type', 'movie')])
```

This is more likely 480p that someone entered the resolution improperly.  Returning as `480x720` would allow further validation on the user's end.

As an extension to this an `aspect_ratio` property could also be generated and if the value is less than `1:1` it can be assumed to be invalid.

**Comment by Toilal** [comment]
Computing an aspect-ratio to perform validation on `0000x0000` `screenSize` pattern is a good idea.

I'll implement it ASAP :)

**Comment by ratoaq2** [comment]
Fixed by #479
